### PR TITLE
803:Migrate from valispace

### LINKED ISSUE #803: Migrate requirements from Valispace into github

### PR DESCRIPTION
**Pull request by dheater**
# Description

Migrate requirements from Valispace into Sphinx generated documents.

Fixes #803 

Updates #1082

Creates #1084, #1085, #1087, and #1088

Note: These requirements are imported from Valispace. I make no certification that they are correct. If you see issues, please open a new ticket against the requirement.

Note that I did not pull deleted/archived requirements from Valispace. This 
# General checklist:

- [x] I have performed a self-review, including - looked through the `Files changed` tab, browsed repository in my branch
- [x] I have made corresponding changes to the documentation - to reflect changes in code, electrical or mechanical design
- [x] All new documentation or graphics follow the [documentation style guide](https://github.com/RespiraWorks/Ventilator/wiki/Documentation-style-guide)
- [x] All new content is linked, easily discoverable, does not require too many clicks
- [x] I have reviewed any other relevant parts of our [contributor wiki](https://github.com/RespiraWorks/Ventilator/wiki) to ensure that this PR conforms to the standards
- [-] I have given the PR a descriptive name (prefixed with **WIP** if it is not yet ready for review)
- [x] I have tagged relevant reviewers

### CODE REVIEW

**Review by martukas — CHANGES_REQUESTED** [review]
Contents look good, but actually I would have liked a link to the generated docs somewhere so I could see what this results in? I hunted around old PRs and on slack, but I could not find it.

Which actually brings me to my next point -- maybe this is already the time to update the `regulatory` dir of the repo to point to this our newly generated github pages rather than valispace? If there is non-redundant stuff in the old generated PDFs, we can keep those links, but at least the new stuff should be discoverable.

**Comment by dheater** [comment]
> Contents look good, but actually I would have liked a link to the generated docs somewhere so I could see what this results in? I hunted around old PRs and on slack, but I could not find it.

To get a viewable local view, you would need to generate locally. That should be doable using the instructions in the README (that would be a good test of the documentation): https://github.com/RespiraWorks/Ventilator/blob/803-migrate-from-valispace/docs/README.md

> Which actually brings me to my next point -- maybe this is already the time to update the `regulatory` dir of the repo to point to this our newly generated github pages rather than valispace? If there is non-redundant stuff in the old generated PDFs, we can keep those links, but at least the new stuff should be discoverable.

I hadn't noticed the `requirements` directory. Should we just delete that?

**Comment by dheater** [comment]
> I hadn't noticed the `requirements` directory. Should we just delete that?

I created a new `process` section in the generated documentation and pulled the still relevant contents from the `requirements` directory. Deleted the old PDF and Excel files with old requirements.

File: docs/process/requirements_management.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
+.. requirements_management:
```
Comment (dheater) [inline docs/process/requirements_management.rst:1]: I didn't write this content. I just migrated it from the old `requirements directory` and updated the links and formatting for Sphinx.

**Review by martukas — APPROVED** [review]
I can definitely think of ways to improve this, but won't require mind reading now. I think there is nothing to block this. Go for it! :)

**Pull request merged**